### PR TITLE
Replace manual pattern checking with jquery validator

### PR DESCRIPTION
**Pull request by tonmoy18**
There is a option for pattern matching in HTML5, i have enabled that, no JS required any more
If some browser does not support HTML5, we can use jquery validator (it is already included). I have utilized it on the entire collection form

### CODE REVIEW

File: app/assets/javascripts/datarepo/funder.js [hunk 1]
```diff
@@ -12,25 +12,23 @@ Blacklight.onLoad(function() {
...
-  });
+$(".simple_form").validate({
+	errorPlacement: function(error, element){		
+		if (element.parent().hasClass("input-group") )		
+			error.insertAfter( element.parent() );
+		else
+			error.insertAfter( element );
+	}
+});
+$(funderClass).rules("add", {datarepoFunder: "[a-zA-Z]: [0-9]"});
```
Comment (yinlinchen) [inline app/assets/javascripts/datarepo/funder.js]: Is funder name always combination of alphabet?  Is award number always combination of numbers?

Comment (tonmoy18) [inline app/assets/javascripts/datarepo/funder.js]: I was confused about that. I can change both to .*

**Comment by tonmoy18** [comment]
fixed pattern issue and squashed the commits.

**Comment by yinlinchen** [comment]
+1

**Pull request merged**